### PR TITLE
Geocat v4.2.3 report custom Google analytics only enabled in PROD

### PR DESCRIPTION
**Pull request by cmangeat**

### CODE REVIEW

File: web/src/main/webapp/xslt/base-layout.xsl [hunk 1]
```diff
@@ -41,14 +41,42 @@
...
+                                        $discoveryServiceRecordUuid,
+                                        'resourceAbstractObject',
+                                        $lang)
+                              else if (contains($nodeName, '|'))
+                              then substring-after($nodeName, '|')
+                              else $nodeName"/>
+
+        <xsl:variable name="actualHtmlHeadDescription" select="if ($isProd) then $htmlHeadDescription else ''"/>
+
+        <title><xsl:value-of select="$htmlHeadTitle"/></title>
```
Comment (sebr72) [inline web/src/main/webapp/xslt/base-layout.xsl]: Need to revert tiltle for INT and DEV

File: web/src/main/webapp/xslt/base-layout.xsl [hunk 2]
```diff
@@ -41,14 +41,42 @@
...
       <head>
-        <title>
-            <xsl:value-of select="util:getNodeName('', $lang, true())"/>
-        </title>
+        <xsl:variable name="discoveryServiceRecordUuid"
+                      select="util:getDiscoveryServiceUuid('')"/>
+        <xsl:variable name="nodeName"
+                      select="util:getNodeName('', $lang, true())"/>
+
+        <xsl:variable name="htmlHeadTitle"
```
Comment (sebr72) [inline web/src/main/webapp/xslt/base-layout.xsl]: start with ga

File: web/src/main/webapp/xslt/base-layout.xsl [hunk 3]
```diff
@@ -41,14 +41,42 @@
...
+                                        $lang,
+                                        $discoveryServiceRecordUuid,
+                                        'resourceTitleObject',
+                                        $lang)
+                              else if (contains($nodeName, '|'))
+                              then substring-before($nodeName, '|')
+                              else $nodeName"/>
+
+
+        <xsl:variable name="htmlHeadDescription"
```
Comment (sebr72) [inline web/src/main/webapp/xslt/base-layout.xsl]: start with ga

**Comment by cmangeat** [comment]
lkhlkhlkhlhlh

**Review by sebr72 — APPROVED** [review]
Looks good. Thanks

**Pull request merged**